requirements.tex

```diff
@@ -64,3 +78,10 @@ \subsection{Release Requirements for non-Operational Project Activities} \label{
 In some cases however, it is necessary to use non-released software, such as release candidates or stable builds.
 In all cases, the software used must be clearly identified (Github revision at least), and the distribution/deployment strictly controlled.
 
+
+\subsection{Assessment} \label{sec:assessment}
+
+As a first glance, it seems  that major releases need to be done more on a feature or milestone bases. 
+
+However before this assessment need further evaluation and clearly identify which of the above requirements is applicable for each of the software products  in appendix \ref{sec:swptree}.
+This extra information shall be included in MagicDraw and documented in \citeds{DMTN-104}, still draft.
```

Comment:
However before concluding, further evaluation is needed to clearly identify which of the above requirements is applicable for each software product listed  in appendix \ref{sec:swptree}.
This extra information shall be included in MagicDraw and documented in \citeds{DMTN-104}, which still draft.